contracts/cm.yaml

```diff
@@ -256,6 +256,59 @@ paths:
             application/xml:
               schema:
                 $ref: '#/components/schemas/ErrorResponse'
+  /v0.5/links/context/notify:
+    post:
+      tags:
+        - hip facing
+        - data notification
+      parameters:
+        - $ref: "#/components/parameters/authorization"
+      summary: API for HIPs to notify CM about new health data added to already linked care contexts
+      description: >
+        This API is called by HIP only when there is new health data is added/created for a patient and under a care context that is already linked with patient's Health Account. 
+        HIP can send following things in this API to notify the Consent Manager about the new health data added:
+          1. **Patient's Identifier for which the new health data is added (It can be NDHM id or NDHM number)**
+          2. **Care Context reference under which the new health data is added**
+          3. **Patient's reference (An identifier with which the patient is registered on HIP)**
+          4. **Types of health information documents that have been added**
+          5. **A date when the health information was created/added on the HIP**
+
+        Note: This API shouldn't be called if the new heath data of is added/created under new care context. 
```

Comment:
Can we give ref to other API in description?
I think let's keep the linking process separate from this. Because then there are multiple ways of linking care context.

Reply:
ok